bibliopixel/drivers/SPI/WS2801.py

```diff
@@ -0,0 +1,19 @@
+from .. channel_order import ChannelOrder
+from . base import SPIBase
+from ... import gamma
+
+
+class WS2801(SPIBase):
+    """Main driver for WS2801 based LED strips on devices like the Raspberry Pi and BeagleBone"""
+
+    def __init__(self, num, gamma=gamma.WS2801, **kwargs):
+        print("Loading WS2801")
+        spi_speed = kwargs.get('spi_speed', 1)
```

Comment:
Yup... that was old from when I first started reworking all the SPI stuff and I was removing all the extraneous crap and letting kwargs handle it. Fixed.